src/config/inversion-fixes.config

```diff
@@ -302,6 +302,14 @@ INVERT
 [aria-label^="Google Account"]
 div[title="Profile"]
 
+CSS
+.kix-embeddedobject-view svg {
+filter: invert(100%) hue-rotate(180deg) !important
```

Comment:
Please add these selectors under `INVERT` rule, it will produce the same CSS code.